charts/internal/calico/templates/node-cpva/psp-gardener-node-cpva.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: {{ include "podsecuritypolicyversion" .}}
+kind: PodSecurityPolicy
+metadata:
+  name: gardener.cloud.node-cpva
```

Comment:
For the sake of #15 I guess it makes sense to prefix these things with `networking.gardener.cloud:calico:` (other resources as well)?

Reply:
I will file another PR just for this.